evalml/tests/dependency_update_check/minimum_core_requirements.txt

```diff
@@ -3,6 +3,7 @@ pandas==1.2.1
 scipy==1.3.3
 scikit-learn==0.24.0
 scikit-optimize==0.8.1
+pyzmq==20.0.0
```

Comment:
I guess we must have missed this in #2389. The confusing thing is that was merged 5 days ago, so why is this showing up now?

Reply:
I think its showing up now because the minimum dependency checker was failing 5 days ago, when `pytest-timeout` was added without a minimum version. 
- https://github.com/alteryx/evalml/pull/2374/files